src/platform/site-wide/side-nav/helpers.js

```diff
@@ -135,9 +144,14 @@ export const normalizeSideNavData = data => {
   // Derive properties we need from data.
   const items = get(data, 'links');
 
+  // Derive only nav items that are relevant for the current page.
+  const relevantNavItems = filter(items, item =>
+    includes(window.location.pathname, get(item, 'url.path')),
```

Comment:
we should use the built in array functions for these instead of lodash includes and filter.

Reply:
also please use optional chaining instead of get

Reply:
Alright 👍